app/src/main/res/values/strings.xml

```diff
@@ -273,8 +274,12 @@
     <string name ="zone_selection_button_cancel">Cancel</string>
     <string name ="zone_selection_download_error">Error downloading zone</string>
 
-
-
+    <!-- Connectivity Alert Dialog -->
+    <string name="connectivity_alert_offline_title">No Internet Connection</string>
+    <string name="connectivity_alert_offline_message">Do you want to use your Offline Maps</string>
```

Comment:
If this message is supposed to be a question, then a question mark is missing. Same for the online one.

Reply:
You're right